css/style.css

```diff
@@ -1,8 +1,248 @@
-/**
- * Add your custom styles below
- * 
- * Remember: 
- * - Be organised, use comments and separate your styles into meaningful chunks
- *    for example: General styles, Navigation styles, Hero styles, Footer etc.
- * 
- */
+*{
+    /* background-color: rgb(250, 231, 234); */
+    text-decoration: none;
+    list-style: none;
+    box-sizing: border-box;
+}
+body {
+    background-color: rgb(250, 217, 222);
+}
+
+/* header */
+
+
+
+
+.nav {
+    display: flex;
+    margin-top: 30px;
+}
+.navbar {
+    padding: 30px;
+}
+.nav li {
+    margin-right: 15px;
+}
+
+.eli {
+    font-weight:bolder;
+    font-size: 30px;
+    color: rgb(126, 82, 90);
+}
+.elirose {
+    color: pink;
+}
+.head {
+    background-image: url("/images/pexels-mo-eid-10035858.jpg");
+    background-size: cover;
+    width: 100%;
+    height: 500px;
+    border-radius: 5px;
+    }
+.title {
+    padding-top: 130px;
+    margin-left: 50px;
+    font-size: 32px;
+    
+}
+
+/* body */
+.part1 {
+    width: 90%;
+    height: auto;
+    border:1px solid rgb(186, 185, 185);
+    margin-top: 20px;
+    background-color: rgb(250, 231, 234);
+    border-radius: 5px;
+}
+   .acu {
+    padding: 20px;
+   } 
+   .acu-img {
+    width: 100%;
+   }
+   .part2 {
+    width: 90%;
+    height: auto;
+    border:1px solid rgb(186, 185, 185);
+    margin-top: 20px;
+    background-color: rgb(250, 231, 234);
+    border-radius: 5px;
+
+   }
+   .acu2 {
```

Comment:
Indentation from here is not correct. If something is not nested it should be all the way to the left